### PR TITLE
Add typescript for Realm Settings page

### PR DESCRIPTION
**Pull request by davidebriani**
This PR adds Typescript's types to Realm Settings page.

### CODE REVIEW

File: src/react/ui/SingleCardPage.tsx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { Container } from 'react-bootstrap';
 import BackButton from './BackButton';
 
 interface Props {
-  backLink: string;
```
Comment (Pavinati) [inline src/react/ui/SingleCardPage.tsx:24]: i saw this line in many other PRs. you should make a separate commit for it so you will have an easier time when rebasing everything after a merge

**Review by Pavinati — APPROVED** [review]

**Review by bettio — APPROVED** [review]
